neqo-transport/src/packet.rs

```diff
@@ -277,7 +285,8 @@ impl PacketBuilder {
         encoder.encode(&[0; 4]); // Zero version == VN.
         encoder.encode_vec(1, dcid);
         encoder.encode_vec(1, scid);
-        encoder.encode_uint(4, QUIC_VERSION);
+        encoder.encode_uint(4, QuicVersion::Draft27.as_u32());
+        encoder.encode_uint(4, QuicVersion::Draft28.as_u32());
```

Comment:
I think that we probably want QuicVersion::iter() or something similar so that we can support this use case more broadly.  This might be the only mainline code that uses it, but we might need to use that facility in testing.

Reply:
opened #693.